pwa/components/common/text/list.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+
+type listPropsType = React.DetailedHTMLProps<React.HTMLAttributes<HTMLUListElement>, HTMLUListElement>;
+export const List: React.FC<listPropsType> = ({ ...props }) => <ul {...props}>{props.children}</ul>;
```

Comment:
With your given code, props already contains the children property so you could do `<ul {...props} />` but that's not what I expect for this component.

Reply:
https://github.com/ReyalWesley/httpCache/pull/24/files#r1228040519